tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -2,7 +2,9 @@
 # enabled. These resources are merged into mock_cai_resources.dump by the
 # update_cai_dumps.py script. Any lines that start with a '#' are ignored.
 #
-# Bigquery Dataset
+# Bigquery Dataset with project id
+{"name":"//bigquery.googleapis.com/projects/project3/datasets/bq_test_ds1","asset_type":"google.cloud.bigquery.Dataset", "resource":{"version":"v2","discovery_document_uri":"https://bigquery.googleapis.com/$discovery/rest","discovery_name":"Dataset","parent":"//cloudresourcemanager.googleapis.com/projects/project3","data":{"creationTime":"1540220016171","datasetReference":{"datasetId":"bq_test_ds1","projectId":"project3"},"location":"us","id":"project3:bq_test_ds1","kind":"bigquery#dataset","lastModifiedTime":"1540220016171"}}}
```

Comment:
Parent needs to be //cloudresourcemanager.googleapis.com/projects/1043 to represent project3.